.github/ISSUE_TEMPLATE/Standard-Feature-Issue.md

```diff
@@ -1,50 +0,0 @@
-## Story
-
-**As a < Agent, Admin, or Patient> of <STDcheck, Healthlabs, UTITreatment and/or Manage >**, I need < some functionality expressed as the user of the app would express the need at a high level >.
-
-I need < a deeper description of the need if necessary (most of the time it will be) >
-
-so that < why do you need this (short description of the business value >.
-
-## Business Value  
-
-< Complete description of the business value - why do we need to make this change - for the requested item. It is useful to have this even for bugs that we feel “must” be fixed because it helps prioritization. >
-
-## Acceptance Criteria
-< a numbered list of behaviors that must be present in order to satisfy the required functionality described above. >
- 
-## Notes
-< bulleted list of special notes or suggestions to the development team on the implementation of the changes. >  
-
-## Definition of Ready
```

Comment:
What's the reason for dropping this and all following sections from the template? Are we still going to add them when we groom the issue or are we switching to some other process to ensure these are addressed?